docs/conf.py

```diff
@@ -10,12 +10,16 @@
 #
 # All configuration values have a default; values that are commented out
 # serve to show the default.
+from __future__ import absolute_import
+from __future__ import unicode_literals
 
-import sys, os
+import os
+import sys
+
+import tron
 
 sys.path.insert(0, os.path.abspath('..'))
```

Comment:
This may have been above the "import tron" for a reason, for doc generation on pythonhosted.org?
Double check to make sure docs generation is not broken after this... I don't know how to do that off the top of my head